api/src/pcapi/core/offers/models.py

```diff
@@ -121,7 +121,7 @@ class Product(PcObject, Base, Model, HasThumbMixin, ProvidableMixin):
     isGcuCompatible: bool = sa.Column(sa.Boolean, default=True, server_default=sa.true(), nullable=False)
     isNational: bool = sa.Column(sa.Boolean, server_default=sa.false(), default=False, nullable=False)
     isSynchronizationCompatible: bool = sa.Column(sa.Boolean, default=True, server_default=sa.true(), nullable=False)
-    last_30_days_booking = sa.Column(sa.Integer, nullable=False, default=0)
+    last_30_days_booking: int = sa.Column(sa.Integer, nullable=False, default=0)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏A fixup dans le commit précédent. 
Et si on est nullable, plus besoin du type hint